foundation/application.go

```diff
@@ -206,3 +225,34 @@ func setRootPath() {
 
 	support.RootPath = rootPath
 }
+
+func getEnvPath() string {
+	envPath := ".env"
+	args := os.Args
+	for index, arg := range args {
+		if strings.HasPrefix(arg, "--env=") {
+			path := strings.TrimPrefix(arg, "--env=")
+
+			if path != "" {
+				envPath = path
+				break
+			}
+		}
+		if strings.HasPrefix(arg, "-env=") {
+			path := strings.TrimPrefix(arg, "-env=")
+
+			if path != "" {
+				envPath = path
+				break
+			}
+		}
+		if arg == "--env" || arg == "-env" {
```

Comment:
Support `--env .env.test`

Reply:
usually  single hyphen ('-') for single-character. 
```bash
 --help -> -h
 --env -> -e?
```

Reply:
Good idea 👍